arm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  elbow1_1_controller:
    type: effort_controllers/JointPositionController
    joint: elbow1_1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  elbow1_2_controller:
    type: effort_controllers/JointPositionController
    joint: elbow1_2
    pid: {p: 100.0, i: 50.01, d: 10.0}
  elbow2_1_controller:
    type: effort_controllers/JointPositionController
    joint: elbow2_1
    pid: {p: 100.0, i: 50.01, d: 10.0}
  elbow2_2_controller:
    type: effort_controllers/JointPositionController
    joint: elbow2_2
    pid: {p: 100.0, i: 50.01, d: 10.0}
  elbow3_1_controller:
    type: effort_controllers/JointPositionController
    joint: elbow3_1
    pid: {p: 100.0, i: 50.01, d: 10.0}
